packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/Chatbot/Chatbot.md

```diff
@@ -17,24 +17,30 @@ propComponents: [
 'ChatbotWelcomePrompt',
 'WelcomePrompt'
 ]
+sortValue: 2
 ---
 
 import Chatbot, { ChatbotDisplayMode } from '@patternfly/virtual-assistant/dist/dynamic/Chatbot';
 import ChatbotWelcomePrompt from '@patternfly/virtual-assistant/dist/dynamic/ChatbotWelcomePrompt';
 
 ### Container
 
-The `Chatbot` component is the chatbot container which adapts to various display modes, as well as both light and dark themes.
+The PatternFly chatbot is anchored to the bottom of the screen, in a separate window that overlays other content in the UI. This container can be shown and hidden via [the chatbot toggle.](/patternfly-ai/chatbot/chatbot-toggle)
```

Comment:
It's technically not always anchored to the bottom - overlay isn't, and embedded is not always. (Embedded is not in this branch.)

Reply:
I changed to "The PatternFly chatbot is a separate window that overlays or is embedded within other UI content." Do you think that's accurate or still missing something?

Reply:
That's perfect!